deploy/examples/create-external-cluster-resources.py

```diff
@@ -1528,6 +1528,9 @@ def init_topology_rbd_pools(self, topology_rbd_pools):
         for pool in topology_rbd_pools:
             self.init_rbd_pool(pool)
 
+    def getCommand(self):
```

Comment:
`getCommand` makes me think this is getting some sort of command itself. Let's use a clearer name like `getScriptCliFlags` so it is clear what the function is doing.

```suggestion
    def getScriptCliFlags(self):
```